### DISCUSSION TITLE
Lesson 6: Script directory error with DeploySimpleStorage

### DISCUSSION THREAD

**Discussion by Jizzie10** [Cyfrin/foundry-full-course-cu#1392] [General]
I'm getting an error while trying to run the script DeploySimpleStorage

`````
Error: 
The project doesn't have any input files. Make sure the `script` directory is configured properly in foundry.toml. Otherwise, provide the path to the file.
`````

Despite adding the script directory to the foundry.toml it still gives the error and it also gives an error while trying to use the "--help" options.

`````
zsh: command not found: --help
`````

What could be the issue?

**Comment by heheboii11** [comment]
share the command you used for running the script

**Reply by Jizzie10** [reply]
I was able to resolve it, so i configured the settings (.zsh_profile) and stored the path of the file in it.
